0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -20,11 +20,13 @@ This spec depends on the [ILP spec](../0003-interledger-protocol/).
 | | [**disconnect**](#disconnect) ( ) `⇒ Promise.<null>` |
 | | [**isConnected**](#isconnected) ( ) `⇒ Boolean` |
 | | [**getInfo**](#getinfo) ( ) <code>⇒ Promise.&lt;[LedgerInfo](#class-ledgerinfo)></code> |
+| | [**getPrefix**](#getprefix) ( ) `⇒ Promise<String>` |
+| | [**getAddress**](#getaddress) ( ) `⇒ Promise<String>` |
```

Comment:
`getAddress` should be `getAccount` imo, see https://github.com/interledger/rfcs/pull/62/commits/6e6851e26897b6d23d64eee723b9a334a4d3d28a#r73805565